### PR TITLE
kelly/caps

### PR DESCRIPTION
**Pull request by kellybinary**
remove all capitalization

### CODE REVIEW

**Comment by ashkanx** [comment]
Please fix the conflicts

**Comment by negarn** [comment]
@kellybinary @binary-hamedanchi I think we can remove all instances of `text-transform`, no?

File: src/javascript/binary/websocket_pages/trade/price.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ const Price = (function() {
                     elementTextContent(h4, Content.localize().textSpreadTypeShort);
                 }
             } else {
-                elementTextContent(h4, display_text);
+                elementTextContent(h4, display_text.replace(/\b\w/g, l => l.toUpperCase())); // capitalise each word
```
Comment (negarn) [inline src/javascript/binary/websocket_pages/trade/price.js]: I think this will break translations?

Comment (kellybinary) [inline src/javascript/binary/websocket_pages/trade/price.js]: @negarn any ideas on how we can capitalised this? `display_text` is lowercased, but in the UI, we need it to be capitalised. maybe we can use css text-transform ?

**Pull request closed without merging**